source/pug/templates/base-for-django.pug

```diff
@@ -0,0 +1,18 @@
+//-
+  This template is used to generate HTML for usage in Mezzanine.
+  There are {{ TOKENS }} in place for Mezzanine to target with CMS data.
+  `npm build` will compile this to `/dest/opportunity/template/index.html`
```

Comment:
is this line true for campaign pages too?

Reply:
Ah I need to update this...not quite accurate